jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/skaffold/FilesMojoV2.java

```diff
@@ -67,6 +72,48 @@
 
   @VisibleForTesting static final String GOAL_NAME = "_skaffold-files-v2";
 
+  // extracting source directories based on https://kotlinlang.org/docs/reference/using-maven.html
+  @VisibleForTesting
+  static Set<Path> getKotlinSourceDirectories(MavenProject project) {
+    Plugin kotlinPlugin = project.getPlugin("org.jetbrains.kotlin:kotlin-maven-plugin");
+    if (kotlinPlugin == null) {
+      return Collections.emptySet();
+    }
+
+    Path projectBaseDir = project.getBasedir().toPath();
+
+    Predicate<PluginExecution> noTestCompileGoal =
```

Comment:
could we just directly include this where the filter is? Instead of defining it separately here.